_posts/en/newsletters/2020-10-14-newsletter.md

```diff
@@ -0,0 +1,175 @@
+  (see [Newsletter #72][news72 upfront payments]).
+
+    This week on the Lightning-Dev mailing list, a
+    [discussion][teinturier dynamic] about a proposed minor protocol
+    change transformed into a conversation about upfront fees:
+
+    - *Incremental routing:* ZmnSCPxj [described][zmnscpxj tunneling]
+      a nested encrypted tunneling protocol where routes would be
+      built incrementally, with the spender being able to pay each
+      successive routing hop independently.  This would ensure routing
+      fees couldn't be stolen by an earlier hop that deliberately failed
+      a route.  A downside to this approach is that it would require a
+      significant number of network round trips, which could make even
+      successful payments take a long time.  A spy node who kept track
+      of routed message timing and HTLC duration could also build rough
+      estimates about how many hops away the spender or receiver are
+      from it, reducing the amount of privacy users would receive from
+      LN.
+
+    - *Trusted upfront payment:* Antoine Riard [promoted][riard trust]
+      the idea of simply paying the upfront fees and, if your peer
+      steals them, downgrade their score so that your routing
+      algorithm prefers not to use them again.  Assuming a single
+      upfront fee is much smaller than the aggregate amount of fees a
+      peer could earn over weeks or months of routing, there should be
+      an incentive to behave honestly.
+
+- **Bech32 addresses for taproot:** Rusty Russell [resumed][russell
+  bech32] a previous discussion (see [Newsletter #107][news107 bech32])
+  about revising the [BIP173][] rules for [bech32][topic bech32]
+  addresses in order to prevent the [bech32 extension bug][] from
+  affecting users of [taproot][topic taproot] and future similar
+  upgrades.  Russell proposed using a backwards-incompatible format
+  based on a [revised bech32 encoding scheme][wuille new bech32]
+  previously described by Pieter Wuille.  This would eliminate the bug
+  but also require wallets upgrade in order to be able to pay taproot
+  users.
+
+    A [previously-proposed alternative][news107 bech32] would be a
+    backwards compatible restriction on bech32 address lengths.  This
+    only directly provides safety to taproot users receiving payments
+    from spenders who upgraded their wallets to enforce the new length
+    restrictions.  In the discussion, it was proposed that broader
+    safety could be provided by also enforcing length restrictions at
+    either the [consensus layer][harding bech32] or the [transaction
+    relay policy layer][o'connor bech32].
+
+    Russell concluded his post by noting that "the sooner a decision is
+    reached on this, the sooner we can begin upgrading software for a
+    taproot world."  Feedback would be especially appreciated from
+    authors of wallets and bitcoin-sending services as they will be
+    asked to implement whatever change is decided upon.
```

Comment:
maybe s/asked/required/

Reply:
There's no requirement that a wallet be able to pay taproot addresses, so it's strictly *asked*.